File: SamplePager/app/src/main/java/org/texchtown/pager/MainActivity.java
package org.texchtown.pager;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;
import androidx.viewpager.widget.PagerAdapter;
import androidx.viewpager2.adapter.FragmentStateAdapter;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import org.w3c.dom.Text;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public static class MyAdapter extends PagerAdapter {

        //layoutinflator 사용을 위한 context
        private Context mContext = null;

        //MyAdapter 생성자 만들기
        public MyAdapter() {

        }
        public MyAdapter(Context context) {
            mContext = context;
        }


        @NonNull
        @Override
        public Object instantiateItem(@NonNull ViewGroup container, int position) {
            View view1 = null;

            if(mContext != null) {
                LayoutInflater inflater = (LayoutInflater) mContext.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
                view1 = inflater.inflate(R.layout.page, container, false);
                TextView textView = (TextView) view1.findViewById(R.id.title);
                textView.setText("Text"+position);
            }
            container.addView(view1);

            return super.instantiateItem(container, position);
        }

        @Override
        public void destroyItem(@NonNull ViewGroup container, int position, @NonNull Object object) {

            container.removeView((View) object);
        }

        @Override
        public int getCount() {
            return 10;
        }

        @Override
        public boolean isViewFromObject(@NonNull View view, @NonNull Object object) {
            return (view == (View) object);
        }

    }
}

File: FlightGame/app/src/main/java/org/texchtown/flightgame/MainActivity.kt
package org.texchtown.flightgame

import android.content.Intent
import android.content.SharedPreferences
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.WindowManager
import org.texchtown.flightgame.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        //set full screen
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN)

        setContentView(binding.root)

        binding.play.setOnClickListener {
            val intent= Intent(this, GameActivity::class.java)
            startActivity(intent)
        }

        val pref: SharedPreferences = getSharedPreferences("game", MODE_PRIVATE)
        binding.highScoreTxt.text = pref.getInt("highscore", 0).toString()
    }

}